### PR TITLE
Added support for google analytics

### PR DESCRIPTION
**Pull request by aashish-jain**
Task link : https://trello.com/c/XFl0wTdk/21-add-google-analytics-to-the-web-page

### CODE REVIEW

**Comment by LinusCastelino** [comment]
This PR requires addition of an environment variable which will be completed as a part of task - https://trello.com/c/PtXUsrcI/28-add-react-env-vars-in-aws-run-scripts

File: src/main/react-front-end/src/model/reducers.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ queue
 import { LOGIN, LOGOUT, PROMOTE, ENDPOINT_PROGRESS, ENDPOINT_UPDATE, UPDATE_HASH } from './actions';
 import { transferOptimizations } from "./actions";
 
+// import {cookie} from "react-cookies";
```
Comment (LinusCastelino) [inline src/main/react-front-end/src/model/reducers.js]: @aashish-jain this can be removed for now.

**Review by LinusCastelino — CHANGES_REQUESTED** [review]

File: src/main/react-front-end/src/model/reducers.js (hunk 1 above)
Comment (aashish-jain) [reply src/main/react-front-end/src/model/reducers.js]: Removed

**Comment by aashish-jain** [comment]
[Add Google Analytics to the web page](https://trello.com/c/XFl0wTdk/21-add-google-analytics-to-the-web-page)

**Review by KiranPrabhakar135 — APPROVED** [review]
Good to merge.

**Review by yinyifu — APPROVED** [review]

File: src/main/react-front-end/src/analytics/ga.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ * Initializes the google analytics library 
+ */ 
+export function initializeReactGA() {
+  if(process.env.REACT_APP_ENV === 'production'){
+    ReactGA.initialize('UA-147970452-1');
+  }
+  else if(process.env.REACT_APP_ENV === 'development'){
+    ReactGA.initialize('UA-147976263-1');    
+  }
+}
```
Comment (LinusCastelino) [inline src/main/react-front-end/src/analytics/ga.js:10]: @aashish-jain we can redo this in just single line. We dont need to store environment identifiers, rather like other env vars we will store the keys and access those.

**Review by atulsam — APPROVED** [review]
Looks good.

**Review by LinusCastelino — APPROVED** [review]
Go ahead!!!

**Pull request merged**